pbcommand/models/common.py

```diff
@@ -176,6 +176,7 @@ class MimeTypes(object):
     CSV = 'text/csv'
     XML = 'application/xml'
     BINARY = 'application/octet-stream'
+    PICKLE = 'application/python-pickle'
```

Comment:
Which pickle protocol are you using? I believe the default is ascii.

Systems that don't natively understand application/python-pickle will drop to  application/octet- stream which may or may not be consistent with the pickle protocol version (i.e., using protocol 2)

Can you clarify?